source/new-starter/prototype/index.html.md.erb

```diff
@@ -74,7 +70,13 @@ To alert developers when they attempt to enter a secret in the codebase, [instal
 
 ## Start the Brexit transition checker and account manager prototype apps
 
-In the command line, go to the `finder-frontend` repo folder and run `govuk-docker-up`. This starts the `finder-frontend` Brexit transition checker and its dependencies.
+To register a new GOV.UK account most users arrive through the Brexit checker.
+Without data from the Brexit checker, the user's dashboard will not show any integrations with GOV.UK services.
+
+You can spin up all apps required to follow the full user registration journey with govuk-docker.
```

Comment:
A quick style point on the use of "can" (apologies if you already know this). We use "can" for optional actions. "Must" is for required actions, and "should" is for recommendations.

Reading this, it seems that you must start these apps to follow the full user registration journey with govuk-docker. Is that correct?

Reply:
As discussed, will swap the sections.
Will amend the line to follow the GOV.UK stye guide for can/must ect.